pkg/kubectl/cmd/expose.go

```diff
@@ -65,6 +65,7 @@ func NewCmdExposeService(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.MarkFlagRequired("port")
 	cmd.Flags().String("type", "", "Type for this service: ClusterIP, NodePort, or LoadBalancer. Default is 'ClusterIP' unless --create-external-load-balancer is specified.")
 	cmd.Flags().Bool("create-external-load-balancer", false, "If true, create an external load balancer for this service (trumped by --type). Implementation is cloud provider dependent. Default is 'false'.")
+	cmd.Flags().String("load-balancer-ip", "", "IP to assign to to the Load Balancer. If empty, an ephemeral IP will be created and used(cloud-provider specific).")
```

Comment:
--ip is pretty ambiguous.  we could do similar to new template flags and say something like 

`--type=LoadBalancer=ip:1.2.3.4`

Now that I wrote it out, I hate it.  I think --ip is too ambiguous.